plugin-build/plugin/src/main/java/ru/kode/android/build/publish/plugin/BuildPublishPlugin.kt

```diff
@@ -221,3 +249,5 @@ private fun buildChangelog(
         }
         .orEmpty()
 }
+
+private fun String.capitalized(): String = replaceFirstChar { it.uppercaseChar() }
```

Comment:
nit: maybe move in StringExts.kt (create if it isn't existed)?
nit: titlecase() is more preferable in general case (but here uppercaseChar is ok also)